ape_solidity/_utils.py

```diff
@@ -91,7 +91,7 @@ def add_entry(self, remapping: ImportRemapping):
         if not str(path).startswith(f".cache{os.path.sep}"):
             path = Path(".cache") / path
 
-        self.import_map[remapping.key] = str(path)
+        self.import_map[os.path.normpath(remapping.key)] = str(path)
```

Comment:
Since we don't officially support Windows or have tests for Windows, you may want to add a comment here explaining why this is needed to lessen the risk that this behavior regresses in the future

Reply:
thanks, added!